tests_e2e/pipeline/scripts/execute_tests.sh

```diff
@@ -10,7 +13,7 @@ WAAGENT_UID=1000
 #
 # Set the correct mode and owner for the private SSH key and generate the public key.
 #
-cd "$HOME"
+cd "$AGENT_TEMPDIRECTORY"
```

Comment:
initially i was re-using the agent vms (currently they are destroyed after each run), so I moved these from $HOME to the temp directory